builtin/commit.c

```diff
@@ -60,15 +60,19 @@ N_("The previous cherry-pick is now empty, possibly due to conflict resolution.\
 "\n");
 
 static const char empty_cherry_pick_advice_single[] =
-N_("Otherwise, please use 'git reset'\n");
+N_("Otherwise, please use 'git cherry-pick --skip'\n");
 
 static const char empty_cherry_pick_advice_multi[] =
-N_("If you wish to skip this commit, use:\n"
+N_("and then use:\n"
 "\n"
-"    git reset\n"
+"    git cherry-pick --continue\n"
 "\n"
-"Then \"git cherry-pick --continue\" will resume cherry-picking\n"
-"the remaining commits.\n");
+"to resume cherry-picking the remaining commits.\n"
+"If you wish to skip this commit, use:\n"
+"\n"
+"    git cherry-pick --skip\n"
+"\n"
+);
```

Comment:
I do like the change to the advice, but would strongly wish for it to be introduced in its own, separate commit. Separate from `cherry-pick --skip`, that is.

Reply:
In addition to this, we should also shift the "documentation" part to this commit.

Reply:
You mean the documentation of the `--skip` option? No, that belongs into the commit that adds the option.

The thing about this here advice is that it is not necessary to run `--skip`. Instead, it benefits from it. Logically, it is a completely separate issue from implementing that option in the first place.